server.go

```diff
@@ -74,6 +74,7 @@ func ProductionServer(srv *http.Server, httpsAddr, domain string, manager *autoc
 
 func Login(m *miniware.Mapper, pgClient *PGClient) func(w http.ResponseWriter, req *http.Request) {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		//w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
This is _not_ the solution to the local emulator access issue. I don't think.

Reply:
kk